### ISSUE TITLE
502 Bad Gateway Error 

### ISSUE THREAD

**Issue by Brandon1419** [Dam998/shopify-monitor#34]
Hello @Dam998 I started the monitor and it ran for 12 hours then it had this error below. Any ideas? Thanks! 

![E6BB1DD9-7BD5-48F7-AE84-6B7008812765](https://user-images.githubusercontent.com/24864274/89820609-4e26ff80-db1b-11ea-8cba-62fdf89d4e7a.jpeg)

**Comment by Brandon1419** [comment]
@Dam998 After restarting it, I then get this error 
<img width="796" alt="Screen Shot 2020-08-10 at 7 16 44 PM" src="https://user-images.githubusercontent.com/24864274/89840374-17161580-db3e-11ea-8cf5-51100a252db3.png">

**Comment by Blvcks** [comment]
What have you got your requestTiming set to, it needs to be minimum of 30000 if you're not using proxies.

**Comment by Brandon1419** [comment]
@Blvcks its at 30000 Which is what it was at by default.

**Comment by Blvcks** [comment]
@Brandon1419 do you have your webhook set correctly?

**Comment by Brandon1419** [comment]
@Blvcks Yes, the monitor works for awhile with my discord but then it disconnects with the error above. Wondering what’s causing it.
